lib/routes.jsx

```diff
@@ -9,26 +9,46 @@ var HeroUnit = require('./hero-unit.jsx');
 var Homepage = require('./homepage.jsx');
 var urls = [];
 
-var Foo = React.createClass({
-  statics: {
-    pageClassName: 'teaching-materials'
-  },
-  render: function() {
-    return (
-      <div>
-        <HeroUnit image="http://placekitten.com/g/1024/480">
-          <h1>I am foo.</h1>
-          <div><Link to="/" className="btn btn-awsm">Meow</Link></div>
-        </HeroUnit>
-        <h2>Content can go here.</h2>
-      </div>
-    );
-  }
-});
+function placeholderPage(options) {
+  return React.createClass({
+    statics: {
+      pageClassName: options.pageClassName
+    },
+    render: function() {
+      return (
+        <div>
+          <HeroUnit image="/img/hero-unit.jpg">
+            <h1>Placeholder: {options.title}</h1>
+          </HeroUnit>
+          <h2>This is a placeholder page for &ldquo;{options.title}&rdquo;.</h2>
+          {options.githubIssue
+           ? <p>Discussion about this page can be found on GitHub at <a
+               href={"https://github.com/mozilla/teach.webmaker.org/issues/" +
+                     options.githubIssue}>
+                 <code style={{
+                   color: '#1F93D0',
```

Comment:
I'm just inlining the style here because it's just a placeholder page that will go away at some point.